egui/src/grid.rs

```diff
@@ -334,7 +332,7 @@ impl Grid {
         // If somebody wants to wrap more things inside a cell,
         // then we should pick a default layout that matches that alignment,
         // which we do here:
-        ui.horizontal(|ui| {
+        ui.allocate_ui_at_rect(ui.cursor(), |ui|{
```

Comment:
The previous code ensured that the default layout of each grid cell is `horizontal`. i.e. if you call `ui.scope(|ui| { ui.label("foo"); ui.label("bar"); })` from within a grid cell, it would show up as "foo bar".

With the new code we will instead use the same layout as whatever the grid sits in (e.g. vertical if it is in a vertical layout). This could break some existing code.

I think the safe choice is

``` rust
ui.allocate_ui_at_rect(ui.cursor(), |ui| {
    ui.horizontal(|ui| {
```